### PR TITLE
Link formatting

### PR DESCRIPTION
**Pull request by Mec-iS**
See #61

### CODE REVIEW

File: hydra_python_core/doc_writer.py [hunk 1]
```diff
@@ -418,48 +504,48 @@ def add_Collection(self, collection: HydraCollection) -> None:
...
             "@type": "EntryPoint",
-
         }
         for item in self.entrypoint.supportedProperty:
             uri = item.id_
             if item.generate() in self.collections:
                 collection_returned = item.generate()
-                collection_id = uri.replace(
-                    "{}EntryPoint".format(DocUrl.doc_url), "{}{}".format(self.url,self.api))
+                collection_id = uri
```
Comment (farazkhanfk7) [inline hydra_python_core/doc_writer.py:527]: Some changes that were made in HydraEntrypoint class in previous commits are resulting in breakage.

File: hydra_python_core/doc_writer.py [hunk 2]
```diff
@@ -468,9 +554,15 @@ def __init__(self, collection: HydraCollection) -> None:
...
         if collection.path:
-            self.id_ = "{}EntryPoint/{}".format(DocUrl.doc_url, quote(collection.path, safe=''))
+            self.id_ = "{}EntryPoint/{}".format(
+                DocUrl.doc_url, quote(collection.path, safe="")
+            )
+            self.base_url = DocUrl.doc_url
         else:
-            self.id_ = "{}EntryPoint/{}".format(DocUrl.doc_url, quote(self.name, safe=''))
+            self.id_ = "{}EntryPoint/{}".format(
+                DocUrl.doc_url, quote(collection.path, safe="")
```
Comment (farazkhanfk7) [inline hydra_python_core/doc_writer.py:563]: if `collection.path` isn't available `self.name` should be used.

**Review by farazkhanfk7 — CHANGES_REQUESTED** [review]
Many changes that were made in these commits throughout HydraEntrypoint class are not compatible with develop branch of hydrus. I have tested both hydrus and core repo locally in same virtual environment and I don't think we should merge these as it will result in the failure of tests in hydrus. Still needs many changes.

@sameshl Can you please confirm ?
@Mec-iS Should we split this one big PR into multiple PRs ?

**Comment by Mec-iS** [comment]
thanks for looking into this.
Please proceed as you see fit, I will follow.

**Comment by chrizandr** [comment]
Yes these commits were to be added to core before merging #518 in hydrus. I had mentioned that in the review too. Let's merge these ASAP so hydrus builds don't fail.

**Comment by sameshl** [comment]
@farazkhanfk7 Can you finish the remaining part of this PR so we merge it?

**Comment by farazkhanfk7** [comment]
Yes. I'll do it today itself. Was facing an issue in pep8 tests. I've added a question on slack about it.

**Comment by Mec-iS** [comment]
Changes applied in #81

**Pull request closed without merging**